Snouter.Api/Snouter.Api/appsettings.json

```diff
@@ -2,6 +2,11 @@
   "Database": {
     "ConnectionString": "Server=localhost;Port=5432;Database=nekadruga;User ID=postgres;Password=new_password;"
```

Comment:
When running the project database wasn't programatically created, I had to create the database in PostgreSQL before running the api